### PR TITLE
implementation of navbar for both mobile and desktop version

### PR DESCRIPTION
**Pull request by alexandreLebeau**
links are not available yet but the whole US is finished

### CODE REVIEW

**Review by eloregnier — APPROVED** [review]

File: client/src/components/Navbar/Navbar.scss [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  .bm-item {
+    display: inline-block;
+  }
+
+  /* Styling of overlay */
+  .bm-overlay {
+    background: hsla(0, 0%, 100%, 1);
+  }
+}
+// Desktop version below //
```
Comment (fantinerudent) [inline client/src/components/Navbar/Navbar.scss:102]: no need of 20 lines of comment

File: client/src/components/Navbar/Navbar.scss [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  .bm-burger-bars {
+    display: none;
+  }
+  #react-burger-menu-btn {
+    display: none;
+  }
+  .bm-burger-button {
+    display: none;
+  }
+
```
Comment (fantinerudent) [inline client/src/components/Navbar/Navbar.scss]: please indent your code properly

File: client/src/components/Navbar/Navbar.scss [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  a {
+    color: $secondaryColor;
+    text-decoration: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  /* visual stuff */
```
Comment (fantinerudent) [inline client/src/components/Navbar/Navbar.scss]: comment not usefull or clear

File: client/src/components/Navbar/Navbar.scss [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  ul {
+    font-family: system-ui, sans-serif;
+    font-size: 1.5rem;
+    font-weight: 900;
+    text-transform: capitalize;
+  }
+  .logo_and_nav {
+    background-color: $backgroundColor;
+  }
+  /* main stuff */
```
Comment (fantinerudent) [inline client/src/components/Navbar/Navbar.scss]: same

**Review by fantinerudent — CHANGES_REQUESTED** [review]
indent your code correctly please ( empty line between the class for example)

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**